_posts/2020-12-14-changelog-137.markdown

```diff
@@ -0,0 +1,109 @@
+    * Gray out `cfg`-disabled items in structure view
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6432 -->
+    * Provide name resolution for Cargo features in [`doc(cfg())`](https://doc.rust-lang.org/beta/unstable-book/language-features/doc-cfg.html) attributes
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6466 -->
+    * Handle function arguments under `cfg` attribute during type inference and analyses
+
+
+* Support [`cfg_attr`](https://doc.rust-lang.org/reference/conditional-compilation.html#the-cfg_attr-attribute) attribute in some cases during name resolution and completion.
+  In general, the plugin now knows that `cfg_attr` can be expanded to other attributes, and the following stuff was adapted to use this knowledge:
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6462 -->
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6460 -->
+    * Resolve and complete paths of `path` attribute and traits in `derive` attributes under `cfg_attr`
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6448 -->
+    * Take into account `cfg_attr` attribute during attribute and lint attribute completion
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6461 -->
+    * Resolve procedural macro [attributes](https://doc.rust-lang.org/reference/procedural-macros.html#attribute-macros) under `cfg_attr`
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6445 -->
+    * Add completion for conditional compilation options and operators inside `cfg_attr()` and `doc(cfg())` attributes
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6434 -->
+    * Provide name resolution for Cargo features in conditional attributes like `cfg_attr(docsrs, doc(cfg(feature = "macros")))`
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6440 -->
+    * Extend `Attribute without parentheses` inspection scope to highlight attributes under `cfg_attr`
+
+  Note, the plugin can't evaluate predicates of `cfg_attr` attribute yet
+
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6417 -->
+* Provide a way to fetch actual info about stdlib packages like dependencies,
+  [edition](https://doc.rust-lang.org/edition-guide/editions/index.html),
+  Cargo [features](https://doc.rust-lang.org/cargo/reference/features.html).
+  It should help the plugin understand stdlib structure better.
+  For example, the feature allows providing proper completion and navigation for items defined in `std::os` module.
+  Note, the corresponding feature works since Rust 1.41 and disabled by default for now.
+  You can enable it via `org.rust.cargo.fetch.actual.stdlib.metadata` option in `Experimental Feature` dialog
+  (`Help | Find Action` and type `Experimental features` to open the dialog)
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6332 -->
+* Add `redirect input` option for `Cargo command` run configuration to provide stdin from desired file
```

Comment:
```suggestion
* Add `Redirect input` option for `Cargo` run configuration to provide stdin from the desired file
```

Reply:
Oh, thanks!
I forgot that we renamed our configurations some time ago